website/views/static/community-guidelines.jade

```diff
@@ -81,6 +81,8 @@ block content
         img(class='pull-left', src='/community-guidelines-images/moderators.png', alt='')
         ul(class='pull-left list-unstyled')
           li
+            strong Beffymaroo
```

Comment:
Not sure if this matters, but it's styled all lowercase: beffymaroo

@lemoness?